packs/trello/config.yaml

```diff
@@ -3,7 +3,16 @@
   token: ""
 
   list_actions_sensor:
-    board_id: c39TEFLt
-    list_id: 55e7456df81e21e9b4aea429
-    filter:
-      - createCard
+    #api_key: ""
+    #token: ""
+    lists:
+    - list_id: 55e7456df81e21e9b4aea429
```

Comment:
Also still open question: does it makes sense to include Trello list/board IDs as an examples or it's better to wipe everything that looks like real data?
> Since Trello list and Trello board are private - none can access my lists.

I followed [GitHub sensor config](https://github.com/StackStorm/st2contrib/blob/master/packs/github/config.yaml#L7), which lists repositories (and found it easier to understand the Sensor config).